local.go

```diff
@@ -328,24 +334,7 @@ func evaluate(funcs functions, funcInfo *FunctionInfo, evalType EvaluationType,
 }
 
 // update updates the stats given the new evaluation
-func update(location *Location, optLoc *Location, stats *Stats, funcInfo *FunctionInfo, evalType EvaluationType, iterType IterationType, startTime time.Time) {
```

Comment:
Oh, sorry, the codereview tool cut off some of the function. Makes more sense now.